packages/imperative/src/profiles/src/doc/config/IProfileTypeConfiguration.ts

```diff
@@ -39,6 +39,10 @@ export interface IProfileTypeConfiguration {
      * @memberof IProfileTypeConfiguration
      */
     schema: IProfileSchema;
+    /**
+     * The version for the JSON schema document (not required).
+     */
+    schemaVersion?: string;
```

Comment:
If the version is specific to the schema, would it make sense to store this as a `version` property on the `IProfileSchema` object instead?

Reply:
Thanks for the suggestion - I've moved `schemaVersion -> version` in `IProfileSchema` (see 97373db)